make/toc.mk

```diff
@@ -1,19 +0,0 @@
-
```

Comment:
presume this is removing the make directory

Reply:
yea, entire file got removed